File: C/Algorithms/1217.c
int minCostToMoveChips(int *position, int positionSize) {
  int odds = 0;
  for (int i = 0; i < positionSize; i++) {
    if (position[i] % 2)
      odds++;
  }
  return odds < positionSize - odds ? odds : positionSize - odds;
}

File: Rust/Algorithms/1217.rs
impl Solution {
    pub fn min_cost_to_move_chips(position: Vec<i32>) -> i32 {
        let mut odds = 0;
        let len = position.len() as i32;
        for i in position {
            if i % 2 == 1 {
                odds += 1;
            }
        }
        let evens = len - odds;
        match evens > odds {
            true => odds,
            false => evens,
        }
    }
}
